src/jobs/config/jobConfig.updateExample.json

```diff
@@ -18,7 +18,7 @@
         {
           "actionType": "validate",
           "jobParamsSchema": {
-            "$schema": "https://json-schema.org/draft/2020-12/schema",
+            "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
One downside of Avj is that it needs to know the schema version during import. The default supports draft v6 and v7.